src/main/java/cs/ualberta/cmput402/tictactoe/board/ScoreBoard.java

```diff
@@ -0,0 +1,48 @@
+package cs.ualberta.cmput402.tictactoe.board;
+
+public class ScoreBoard {
+    private int X_win,X_loss,X_tie;
+    private int O_win,O_loss,O_tie;
```

Comment:
X_tie and O_tie are the same variables. You only need one tie variable.

Reply:
We fixed it in commit 815e331.
We removed the redundant variables "X_loss" and "O_loss", and replaced "X_tie" and "Y_tie" with one variable "tie".